rebasehelper/specfile.py

```diff
@@ -270,11 +273,14 @@ def get_archive(self):
         """
         return os.path.basename(self.get_sources()[0])
 
-    def _get_raw_source_string(self, source_num: int) -> Optional[str]:
+    def _get_raw_source_string(self, source_num: Optional[int]) -> Optional[str]:
+        if source_num is None:
+            return ""
```

Comment:
I think it would be better to return `None`, since `SpecFile.get_raw_tag_value()` also returns `None` if there is no matching tag.